test/e2e/specs/wp-my-home-support-search-spec.js

```diff
@@ -38,28 +35,9 @@ describe( `[${ host }] My Home "Get help" support search card: (${ screenSize })
 	step( 'Login and select a non My Home page', async function () {
 		const loginFlow = new LoginFlow( driver );
 
-		// The "inline help" FAB should not appear on the My Home
-		// because there is already a support search "Card" on that
-		// page. Therefore we select the "Themes" page for our tests
-		// and then we will switch away to the Homepage to verify
-		// that the Inline Help is not shown
-		await loginFlow.loginAndSelectThemes();
-
-		// Initialize the helper component for the Inline Popover
-		inlineHelpPopoverComponent = await InlineHelpPopoverComponent.Expect( driver );
-
-		// Check the InlineHelp toggle is actually visible in the first instance...
-		const isToggleVisible = await inlineHelpPopoverComponent.isToggleVisible();
-		assert.equal(
-			isToggleVisible,
-			true,
-			'Inline Help support search was not displayed on a non My Home page.'
-		);
-	} );
-
-	step( 'Select the My Home page', async function () {
-		const sidebarComponent = await SidebarComponent.Expect( driver );
-		await sidebarComponent.selectMyHome();
```

Comment:
Not sure why you're removing this here?

I really don't think you can remove this code without reinstating it a followup PR.

It's really important we validate somewhere that the FAB doesn't appear on My Home. Personally I still see this test as the logical home for this.

Reply:
I felt like it was more appropriate on the FAB, as the FAB and the search component don't really have anything to do with each other, IMO. If there were an issue with the FAB showing up on the `/home` route, it would be a FAB issue, not a support search issue, so I was planning to put it in the FAB related tests.